### PR TITLE
Wiretransfer: after data is imported, update the application state

### LINKED ISSUE #1032: Wiretransfer: after data is imported, update the application state
### Steps to reproduce

1. Create a user and an application in the state `AWAITING_PAYMENT`
2. Run the management comment `import_wiretransfers` with a payment sufficient to complete the payment for the bootcamp

### Expected Behavior

1. Since the payment is complete, the application state should advance to `COMPLETE`
2. Since the state is `COMPLETE` an enrollment object should be created

### Actual Behavior

- application state remains in `AWAITING PAYMENT`
- no enrollment is created

### PR DESCRIPTION
**Pull request by umarmughal824**
#### What are the relevant tickets?
fixes #1032 

#### What's this PR do?
Wiretransfer: after data is imported, update the application state

#### How should this be manually tested?
Run the `import_wire_transfers` management command for `AWAITING_PAYMENT` application

#### Screenshots (if appropriate)
Screenshot 2021-01-26 at 17 11 32

### CODE REVIEW

File: ecommerce/api.py [hunk 1]
```diff
@@ -580,6 +581,22 @@ def import_wire_transfer(wire_transfer, header_row):
...
             order=order,
         )
+
+        if application.is_paid_in_full:
+            application.state = AppStates.COMPLETE.value
+            application.save()
+
+            _, created = BootcampRunEnrollment.objects.get_or_create(
+                user=user, bootcamp_run=bootcamp_run
+            )
```
Comment (mbertrand) [inline ecommerce/api.py:591]: Might be better to replace the above with 

```python
complete_successful_order(order, send_receipt=????)
``` 

Not sure if we'd want to send a receipt to the user or not, Peter probably knows.
The above function would also run `novoed_tasks.enroll_users_in_novoed_course.delay` if appropriate.

**Review by mbertrand — CHANGES_REQUESTED** [review]

**Comment by pdpinch** [comment]
> Not sure if we'd want to send a receipt to the user or not, Peter probably knows.
> 
Yes, we want to send a receipt when the wiretransfer is imported — same as if we had completed a Cybersource transaction. 
> The above function would also run novoed_tasks.enroll_users_in_novoed_course.delay if appropriate.
> 
We also want bootcampers to be enrolled in novoed, if appropriate (i.e. if they have completed paying their personal price, and there is a novoed stub set up).

**Review by mbertrand — APPROVED** [review]
👍

File: ecommerce/api.py [hunk 2]
```diff
@@ -580,6 +579,8 @@ def import_wire_transfer(wire_transfer, header_row):
             },
             order=order,
         )
+        complete_successful_order(order, send_receipt=False)
```
Comment (mbertrand) [inline ecommerce/api.py:582]: Based on Peter's comment, can you drop the `send_receipt=False`?

**Review by mbertrand — CHANGES_REQUESTED** [review]

**Review by mbertrand — APPROVED** [review]
👍

**Pull request merged**